src/common/test-runtime/project.json

```diff
@@ -9,7 +9,7 @@
     "System.Runtime.InteropServices.RuntimeInformation": "4.4.0-beta-24813-03",
     "System.Linq.Parallel": "4.0.1",
     "OpenCover": "4.6.519",
-    "ReportGenerator": "2.5.0",
```

Comment:
@safern I'm not sure what this file does: is it superseded by xunit.runtime.depproj in corefx? Nevertheless, changing it here also.

Reply:
this file is old and I don't believe we use it any longer since all our consuming repo's are now using msbuild based CLI.

Reply:
I think to update it in our repos you would have to change the Xunit.Runtime.depproj, not here.

https://github.com/dotnet/corefx/blob/master/external/test-runtime/XUnit.Runtime.depproj#L65

Reply:
I see you already have a PR to do that. So looks good.